gradle/libs.versions.toml

```diff
@@ -1,9 +1,9 @@
 [versions]
 # Project versions
-junitPlatformNative = "0.9.2"
-nativeGradlePlugin = "0.9.2"
-nativeMavenPlugin = "0.9.2"
-utils = "0.9.2"
+junitPlatformNative = "0.9.3-SNAPSHOT"
+nativeGradlePlugin = "0.9.3-SNAPSHOT"
+nativeMavenPlugin = "0.9.3-SNAPSHOT"
+utils = "0.9.3-SNAPSHOT"
```

Comment:
If I saw correctly, you haven't run `updateSamples` task? 
We might want to write a comment here that reminds developer that samples should be kept in sync with this file

Reply:
Actually `master` should probably _never_ use a fixed version but always be `SNAPSHOT`.

Reply:
fixed

Reply:
We need to have one commit that uses a fixed version in order to link to it when we publish a release.
ATM my process is that I create that one release commit, and then immediately push another one with a version bump. I haven't done it now since we have a release delay.
 
We _could_ have that commit on a separate branch. but it seems weird and confusing. I'll push version bumping commit today anyway.

Reply:
Oh you already did it, great!

Reply:
Yes, there needs to be one _commit_, but ideally this should only be pushed on release.